bower.json

```diff
@@ -12,7 +12,8 @@
     "purescript-console": "^3.0.0",
     "purescript-react": "^5.1.0",
     "purescript-generics-rep": "^5.4.0",
-    "purescript-record": "^0.2.6"
+    "purescript-record": "^0.2.6",
+    "purescript-web-mercator": "f-o-a-m/purescript-web-mercator#master"
```

Comment:
Let's tag ps-web-mercator w/ a version to avoid to use every future changes of `master`.